README.md

```diff
@@ -3,6 +3,33 @@
 [![Backend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Backend%20CI/badge.svg)](https://github.com/aggie-coding-club/rev-registration/actions?query=branch%3Amaster+workflow%3A%22Backend+CI%22)
 [![Frontend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Frontend%20CI/badge.svg)](https://github.com/aggie-coding-club/rev-registration/actions?query=branch%3Amaster+workflow%3A%22Frontend+CI%22)
 
+![project-demo](assets/project-demo.gif)
+
+## Description
+
+Rev Registration is an automatic class scheduler for Texas A&M, aimed at easing the process
+of course registration by  generating compatible schedules given the courses a student would
+like to take and what preferences they have. Simply select a term, pick your courses, mark off when
+you're not available, and we'll generate schedules for you!
+
+For instance, imagine you've settled on 3 sections of a course you are fine with having and are having
+trouble finding a schedule without conflicts. Select all 3 under the Section customization level
```

Comment:
```suggestion
trouble finding a schedule without conflicts. Select all 3 under the Section customization level, add in your other classes,
```

Reply:
Great point! I think this is a pretty important distinction tbh